Unix/server/server.c

```diff
@@ -62,7 +61,7 @@ static int _StartEngine(int argc, char** argv, char ** envp, const char *engineS
     Strlcat(engineFile, "/omiengine", PAL_MAX_PATH_SIZE);
     argv[0] = engineFile;
 
-    r = BinaryProtocolListenFile(engineSockFile, &s_data.mux[0], &s_data.protocol0, secretString);
+    r = BinaryProtocolListenFile(engineSockFile, NULL, &s_data.protocol0, secretString);
```

Comment:
I finally understand why this is possible as it means you are put handling the new non-root messages within server itself, rather in the protocol layer itself. This needs to be put back, the code relocated into the server and basically handle strands properly.